hydra_pywr/runner.py

```diff
@@ -629,16 +669,37 @@ def generate_array_recorder_resource_scenarios(self):
                     new_col_names.append(colname.split(':')[1].strip())
                 df.columns = new_col_names
 
+            if "__:" in recorder.name:
+                nodename = recorder.name.split(":")[0].replace("__", "")
+                attrname = recorder.name.split(":")[1]
```

Comment:
This parsing happens often enough that a general func with some error handling could be useful; one approach is [in the pywrparser](https://github.com/pmslavin/pywrparser/blob/master/pywrparser/utils.py#L61).

Reply:
Resolved